crates/bevy_text/src/text.rs

```diff
@@ -103,6 +106,14 @@ impl Text {
         self.alignment = alignment;
         self
     }
+
+    pub const fn with_linebreak_behaviour(
```

Comment:
Can you remove this? We're planning to remove this pattern already.

Reply:
So is this method still OK on the bundles `TextBundle` and `Text2dBundle`? 
If it's alright there, then should I go ahead and add a `with_alignment` to `Text2dBundle` too? Right now the text2d example uses the `with_xxxxx` methods directly on `Text` in building to set these options. 

Otherwise, what would be the preferred way to make these available?

Reply:
Oh, I would suggest removing this pattern from all of them. Cc @Weibye, who's been following that discussion.

The preferred approach is just to set the field's value, and then use `..default()` to populate the rest of the struct.

Reply:
That's easy enough to do then.

Reply:
> That's easy enough to do then. Maybe I should see about removing `with_alignment` or `with_text_alignment` as well? What do you reckon?

Decided it's best not to disrupt anything by doing that.

Reply:
> Maybe I should see about removing with_alignment or with_text_alignment as well? What do you reckon?

The methods on TextBundle will be removed by #5513 but I didn't realize it was also on the Text2DBundle. We should probably just spin up a separate PR that deals with both in one go.

See: https://github.com/bevyengine/bevy/pull/5513/files#diff-f466d9189e476178c2f7a05eac089635d636aac25d72af35e8daae749b11581cL129-L160